### DISCUSSION TITLE
中文从前天开始就是乱码，不知道是什么问题，与账号无关，换了一批账号都是一样的问题，怀疑跟https://chatgpt.duti.tech/这个服务的返回编码有关

### DISCUSSION THREAD

**Discussion by bynhack** [acheong08/ChatGPT#819] [Q&A]
def get_conversations(self, offset=0, limit=20):
        """
        Get conversations
        :param offset: Integer
        :param limit: Integer
        """
        url = BASE_URL + f"api/conversations?offset={offset}&limit={limit}"
        response = self.session.get(url)
        self.__check_response(response)
        print(response.text)
        data = json.loads(response.text)
        return data["items"]
        
在源码的get_conversations打印response的时候就是乱码

{"items":[{"id":"ccd70aff-0b08-45c8-92a2-fb914c6212f7","title":"Há» trá»£ Äáº§u vÃ o.","create_time":"2023-02-19T14:10:56.544748"},{"id":"00c0bacc-6e4e-49d9-b777-34b87066d395","title":"Model recommends tech products.","create_time":"2023-02-19T11:24:54.689251"},{"id":"84ae68e8-9789-4b4e-8a40-5ea3b652f492","title":"Trá»£ giÃºp chatbot.","create_time":"2023-02-19T11:15:51.565798"},{"id":"e152a8e8-37e1-4721-b413-3125a8982149","title":"CDUT User Request","create_time":"2023-02-19T10:40:32.050092"},{"id":"77e4e310-0838-475a-91a3-3cd9e95f0534","title":"Model suggests outfits.","create_time":"2023-02-19T09:46:16.542380"},{"id":"4cfef4cf-0c28-45d1-a8fa-7bf464577664","title":"YÃªu cáº§u há» trá»£.","create_time":"2023-02-19T09:43:45.195652"},{"id":"ce2c25f5-b2eb-4ed4-bc06-792692bb922d","title":"YÃªu cáº§u giÃºp Äá»¡.","create_time":"2023-02-19T08:41:45.864465"},{"id":"9357fe68-5af7-40f5-99af-b72a117c0b0a","title":"YÃªu cáº§u vá» ngoáº¡i hÃ¬nh.","create_time":"2023-02-19T08:41:07.536757"},{"id":"2b7fd2c8-54e7-4dd5-b8b3-3dd34b603098","title":"BÃ i thÆ¡ vá» TÃº.","create_time":"2023-02-19T08:39:55.340033"},{"id":"8b87717c-b670-45ae-a1e6-a2fc5561b22d","title":"å å¼ºä¸­æ¾³åä½","create_time":"2023-02-19T08:22:09.233632"},{"id":"68199eb0-c816-488b-a137-8b4dea580d5c","title":"ä¸­è¶è´¸æé¡ºå·®","create_time":"2023-02-19T07:25:29.881507"},{"id":"f9bd2d24-81fc-4bd3-8eeb-1a482cf85f9d","title":"æ°æ¶ä»£çä¸»é¢ä¹å­ (New Era Theme Parks)","create_time":"2023-02-19T07:17:38.268352"},{"id":"1e4c8d36-24a7-496b-9e75-2e26a0d5fb73","title":"TÄng trÆ°á»ng kinh táº¿ tÆ° nhÃ¢n 2022.","create_time":"2023-02-19T07:16:10.516317"},{"id":"31b67134-4b81-4d84-b3e3-6c83092bc182","title":"TrÃ¡nh nÃ³ng Háº±ng SÆ¡n","create_time":"2023-02-19T07:12:23.034879"},{"id":"fc3745f4-2407-4543-8c79-aa1e72820e9f","title":"Há» trá»£ ngÆ°á»i dÃ¹ng.","create_time":"2023-02-19T07:10:36.055545"},{"id":"d28b3d69-335a-4c13-a8d4-27075bfe4cb2","title":"Há»i vÃ  giÃºp Äá»¡.","create_time":"2023-02-19T07:08:56.356524"},{"id":"f3064270-b3e6-48b2-9209-9ef001d504d1","title":"Há»i vÃ  tráº£ lá»i.","create_time":"2023-02-19T07:08:11.567512"},{"id":"46226974-3495-4432-8ee7-068787197ae7","title":"Há» trá»£ trá»£ lÃ½ áº£o","create_time":"2023-02-19T07:01:31.029240"},{"id":"db4633bb-c45f-42e6-8d54-18bf30ba758b","title":"YÃªu cáº§u vÃ  Tráº£ lá»i.","create_time":"2023-02-19T06:50:36.329757"},{"id":"5b7be755-e90f-487a-a2ca-24ac4837771a","title":"Luxshare Factory in Vietnam.","create_time":"2023-02-19T06:44:19.939705"},{"id":"758d69d1-47d8-4fed-88ea-54d679887045","title":"Äiá»n thoáº¡i hoáº¡t Äá»ng.","create_time":"2023-02-19T06:40:52.601930"},{"id":"d0e36c5d-8365-4c1e-b805-2f9caa91aaa0","title":"BÃ i thÆ¡ vá» Trang","create_time":"2023-02-19T06:38:37.646451"},{"id":"705f753d-6e3d-408a-8852-656e29df1599","title":"Há»i vÃ  tráº£ lá»i.","create_time":"2023-02-19T06:16:40.675114"},{"id":"ede38a01-f3ec-4176-aa06-4705e054735a","title":"ç¨æ·é®é®é¢ï¼æ¨¡ååç­","create_time":"2023-02-18T12:40:26.203789"},{"id":"19be8680-720a-49f7-9192-dcca0f9918f5","title":"ç¨æ·è¯·æ±å¸®å©","create_time":"2023-02-18T12:39:33.963501"},{"id":"f009f556-cde8-42b0-ac76-15231d196c4c","title":"å¨æ°ä¼¦çé³ä¹æå°±","create_time":"2023-02-18T03:37:25.688037"},{"id":"0ed091e0-319a-4fb9-9e9e-1635cdbc2423","title":"ChatGPT å¸®å©ç¨æ·","create_time":"2023-02-18T03:26:51.190148"},{"id":"c40ea1a8-1413-4ea8-98d3-352664cd3bd1","title":"ä½è²ä¿±ä¹é¨ç ç©¶","create_time":"2023-02-18T01:54:04.516060"},{"id":"61605a9b-eaea-4e39-9795-15b299863358","title":"ä½è²ä¿±ä¹é¨ç ç©¶","create_time":"2023-02-18T01:52:26.265229"},{"id":"602a3b1f-503a-45f2-aa60-6b1ef3a6a20d","title":"ç¨Visioç»ç»æå¾","create_time":"2023-02-17T11:38:29.675274"},{"id":"93baf991-8529-44f7-a43d-01f9b5155a54","title":"èªå¨åæ°å¼æ°´æ± è¯éªç³»ç»","create_time":"2023-02-17T11:19:21.074599"},{"id":"f920f328-56a4-48f2-8e37-b544b67c8f67","title":"Data Preprocessing Module.","create_time":"2023-02-17T10:06:25.118011"},{"id":"9198d639-ac1a-4eb7-951e-ff99bb759170","title":"ä½¿ç¨NDKæ¨éè§é¢æµã","create_time":"2023-02-17T09:18:38.777665"},{"id":"7bfa0b45-cecf-4a8d-b81b-6f78d2411511","title":"è°·æ­æµè§å¨æªå¾æä»¶","create_time":"2023-02-17T07:17:03.053739"},{"id":"6a91d776-3ce1-4703-af6c-b4e03eec276a","title":"æ´æ°QOpenGLTextureçº¹ç","create_time":"2023-02-17T07:13:28.160207"},{"id":"fa41e935-4549-4f35-a38f-a59154459ea0","title":"ç¨æ·è¯·æ±å¸®å©","create_time":"2023-02-17T06:25:01.085311"},{"id":"c1671ab3-a9d1-4bd0-b817-8f29e7f64a22","title":"OpenGLè´´å¾æ¯æéæPNG","create_time":"2023-02-17T03:04:40.888138"},{"id":"c65b1753-c917-41f2-a69a-123f56fa1616","title":"å¤çä¸ç¼çç¸","create_time":"2023-02-17T01:11:41.350951"},{"id":"b42a37fd-4649-450a-bbce-64d808135d32","title":"5è½®æ­å¾ååæ¢æé®","create_time":"2023-02-16T15:00:35.446229"},{"id":"4b2f0167-c2b8-4d94-b86e-fd713e85554d","title":"New chat","create_time":"2023-02-16T14:58:04.793101"},{"id":"5f88c68e-2e5c-4eb5-9ae5-c86a7ff7f6fd","title":"çäººçè¯·æ±","create_time":"2023-02-16T07:11:20.680333"},{"id":"1ffe1d2c-827a-42ed-9986-66d40aa2f4fa","title":"å¨æ¼«äººç©å¾çè¯·æ±","create_time":"2023-02-16T04:37:45.703397"},{"id":"cd48a15d-4fd7-4f4e-99d5-12b7274f6400","title":"ChatGPTè½å¸®å°åï¼ (Can ChatGPT Help?)","create_time":"2023-02-16T04:08:43.099001"},{"id":"332ce738-23cf-45ef-9268-8dace9816079","title":"5-image carousel website","create_time":"2023-02-16T04:04:41.115613"},{"id":"7eef9f92-7bbe-4080-9f30-6a4301ff33aa","title":"æºè½æ¨¡åçè½å","create_time":"2023-02-16T04:03:21.610758"},{"id":"4bd8951f-1432-4665-9237-72baa2e94ea6","title":"ç¨æ·å¯»æ±å¸®å©","create_time":"2023-02-16T04:02:55.448398"},{"id":"b451a201-18bc-4c17-90e2-4ce966909882","title":"Android åæ¢è§å¾å±çº§ã","create_time":"2023-02-15T12:21:42.266015"},{"id":"7c36fa57-7b11-4b00-83a4-26031fd4ff8d","title":"è®¡ç®ç§çä¸­çç¹","create_time":"2023-02-15T09:02:25.910541"},{"id":"d2ff0a10-c02c-43e8-8e14-df6f865ecb4d","title":"è®¡ç®ç»çº¬åº¦è·ç¦»åè§åº¦ (Calculate Distance & Angle)","create_time":"2023-02-15T07:49:23.931461"},{"id":"47b4a5ff-94fc-476c-b550-9eb143cd4c11","title":"æ¥ææ¥è¯¢","create_time":"2023-02-15T04:25:55.024948"}],"total":58,"limit":50,"offset":0}

自己的代码是这样调用的：

async def get_conversations(websocket, path):
    async for id in websocket:
        logging.info("查询聊天列表...")
        logging.info(sys.getdefaultencoding())
        try:
            conversations = chatbot.get_conversations()
            await websocket.send(json.dumps(conversations, indent=4))
        except Exception as e:
            logger.error(e)
            await websocket.send(json.dumps({}, indent=2,ensure_ascii=False))
        finally:
            logger.info("消息回复完成")
            
返回的值：

[ { "id": "ccd70aff-0b08-45c8-92a2-fb914c6212f7", "title": "H\u00e1\u00bb\u0097 tr\u00e1\u00bb\u00a3 \u00c4\u0091\u00e1\u00ba\u00a7u v\u00c3\u00a0o.", "create_time": "2023-02-19T14:10:56.544748" }, { "id": "00c0bacc-6e4e-49d9-b777-34b87066d395", "title": "Model recommends tech products.", "create_time": "2023-02-19T11:24:54.689251" }, { "id": "84ae68e8-9789-4b4e-8a40-5ea3b652f492", "title": "Tr\u00e1\u00bb\u00a3 gi\u00c3\u00bap chatbot.", "create_time": "2023-02-19T11:15:51.565798" }, { "id": "e152a8e8-37e1-4721-b413-3125a8982149", "title": "CDUT User Request", "create_time": "2023-02-19T10:40:32.050092" }, { "id": "77e4e310-0838-475a-91a3-3cd9e95f0534", "title": "Model suggests outfits.", "create_time": "2023-02-19T09:46:16.542380" }, { "id": "4cfef4cf-0c28-45d1-a8fa-7bf464577664", "title": "Y\u00c3\u00aau c\u00e1\u00ba\u00a7u h\u00e1\u00bb\u0097 tr\u00e1\u00bb\u00a3.", "create_time": "2023-02-19T09:43:45.195652" }, { "id": "ce2c25f5-b2eb-4ed4-bc06-792692bb922d", "title": "Y\u00c3\u00aau c\u00e1\u00ba\u00a7u gi\u00c3\u00bap \u00c4\u0091\u00e1\u00bb\u00a1.", "create_time": "2023-02-19T08:41:45.864465" }, { "id": "9357fe68-5af7-40f5-99af-b72a117c0b0a", "title": "Y\u00c3\u00aau c\u00e1\u00ba\u00a7u v\u00e1\u00bb\u0081 ngo\u00e1\u00ba\u00a1i h\u00c3\u00acnh.", "create_time": "2023-02-19T08:41:07.536757" }, { "id": "2b7fd2c8-54e7-4dd5-b8b3-3dd34b603098", "title": "B\u00c3\u00a0i th\u00c6\u00a1 v\u00e1\u00bb\u0081 T\u00c3\u00ba.", "create_time": "2023-02-19T08:39:55.340033" }, { "id": "8b87717c-b670-45ae-a1e6-a2fc5561b22d", "title": "\u00e5\u008a\u00a0\u00e5\u00bc\u00ba\u00e4\u00b8\u00ad\u00e6\u00be\u00b3\u00e5\u0090\u0088\u00e4\u00bd\u009c", "create_time": "2023-02-19T08:22:09.233632" }, { "id": "68199eb0-c816-488b-a137-8b4dea580d5c", "title": "\u00e4\u00b8\u00ad\u00e8\u00b6\u008a\u00e8\u00b4\u00b8\u00e6\u0098\u0093\u00e9\u00a1\u00ba\u00e5\u00b7\u00ae", "create_time": "2023-02-19T07:25:29.881507" }, { "id": "f9bd2d24-81fc-4bd3-8eeb-1a482cf85f9d", "title": "\u00e6\u0096\u00b0\u00e6\u0097\u00b6\u00e4\u00bb\u00a3\u00e7\u009a\u0084\u00e4\u00b8\u00bb\u00e9\u00a2\u0098\u00e4\u00b9\u0090\u00e5\u009b\u00ad (New Era Theme Parks)", "create_time": "2023-02-19T07:17:38.268352" }, { "id": "1e4c8d36-24a7-496b-9e75-2e26a0d5fb73", "title": "T\u00c4\u0083ng tr\u00c6\u00b0\u00e1\u00bb\u009fng kinh t\u00e1\u00ba\u00bf t\u00c6\u00b0 nh\u00c3\u00a2n 2022.", "create_time": "2023-02-19T07:16:10.516317" }, { "id": "31b67134-4b81-4d84-b3e3-6c83092bc182", "title": "Tr\u00c3\u00a1nh n\u00c3\u00b3ng H\u00e1\u00ba\u00b1ng S\u00c6\u00a1n", "create_time": "2023-02-19T07:12:23.034879" }, { "id": "fc3745f4-2407-4543-8c79-aa1e72820e9f", "title": "H\u00e1\u00bb\u0097 tr\u00e1\u00bb\u00a3 ng\u00c6\u00b0\u00e1\u00bb\u009di d\u00c3\u00b9ng.", "create_time": "2023-02-19T07:10:36.055545" }, { "id": "d28b3d69-335a-4c13-a8d4-27075bfe4cb2", "title": "H\u00e1\u00bb\u008fi v\u00c3\u00a0 gi\u00c3\u00bap \u00c4\u0091\u00e1\u00bb\u00a1.", "create_time": "2023-02-19T07:08:56.356524" }, { "id": "f3064270-b3e6-48b2-9209-9ef001d504d1", "title": "H\u00e1\u00bb\u008fi v\u00c3\u00a0 tr\u00e1\u00ba\u00a3 l\u00e1\u00bb\u009di.", "create_time": "2023-02-19T07:08:11.567512" }, { "id": "46226974-3495-4432-8ee7-068787197ae7", "title": "H\u00e1\u00bb\u0097 tr\u00e1\u00bb\u00a3 tr\u00e1\u00bb\u00a3 l\u00c3\u00bd \u00e1\u00ba\u00a3o", "create_time": "2023-02-19T07:01:31.029240" }, { "id": "db4633bb-c45f-42e6-8d54-18bf30ba758b", "title": "Y\u00c3\u00aau c\u00e1\u00ba\u00a7u v\u00c3\u00a0 Tr\u00e1\u00ba\u00a3 l\u00e1\u00bb\u009di.", "create_time": "2023-02-19T06:50:36.329757" }, { "id": "5b7be755-e90f-487a-a2ca-24ac4837771a", "title": "Luxshare Factory in Vietnam.", "create_time": "2023-02-19T06:44:19.939705" }, { "id": "758d69d1-47d8-4fed-88ea-54d679887045", "title": "\u00c4\u0090i\u00e1\u00bb\u0087n tho\u00e1\u00ba\u00a1i ho\u00e1\u00ba\u00a1t \u00c4\u0091\u00e1\u00bb\u0099ng.", "create_time": "2023-02-19T06:40:52.601930" }, { "id": "d0e36c5d-8365-4c1e-b805-2f9caa91aaa0", "title": "B\u00c3\u00a0i th\u00c6\u00a1 v\u00e1\u00bb\u0081 Trang", "create_time": "2023-02-19T06:38:37.646451" }, { "id": "705f753d-6e3d-408a-8852-656e29df1599", "title": "H\u00e1\u00bb\u008fi v\u00c3\u00a0 tr\u00e1\u00ba\u00a3 l\u00e1\u00bb\u009di.", "create_time": "2023-02-19T06:16:40.675114" }, { "id": "ede38a01-f3ec-4176-aa06-4705e054735a", "title": "\u00e7\u0094\u00a8\u00e6\u0088\u00b7\u00e9\u0097\u00ae\u00e9\u0097\u00ae\u00e9\u00a2\u0098\u00ef\u00bc\u008c\u00e6\u00a8\u00a1\u00e5\u009e\u008b\u00e5\u009b\u009e\u00e7\u00ad\u0094", "create_time": "2023-02-18T12:40:26.203789" }, { "id": "19be8680-720a-49f7-9192-dcca0f9918f5", "title": "\u00e7\u0094\u00a8\u00e6\u0088\u00b7\u00e8\u00af\u00b7\u00e6\u00b1\u0082\u00e5\u00b8\u00ae\u00e5\u008a\u00a9", "create_time": "2023-02-18T12:39:33.963501" }, { "id": "f009f556-cde8-42b0-ac76-15231d196c4c", "title": "\u00e5\u0091\u00a8\u00e6\u009d\u00b0\u00e4\u00bc\u00a6\u00e7\u009a\u0084\u00e9\u009f\u00b3\u00e4\u00b9\u0090\u00e6\u0088\u0090\u00e5\u00b0\u00b1", "create_time": "2023-02-18T03:37:25.688037" }, { "id": "0ed091e0-319a-4fb9-9e9e-1635cdbc2423", "title": "ChatGPT \u00e5\u00b8\u00ae\u00e5\u008a\u00a9\u00e7\u0094\u00a8\u00e6\u0088\u00b7", "create_time": "2023-02-18T03:26:51.190148" }, { "id": "c40ea1a8-1413-4ea8-98d3-352664cd3bd1", "title": "\u00e4\u00bd\u0093\u00e8\u0082\u00b2\u00e4\u00bf\u00b1\u00e4\u00b9\u0090\u00e9\u0083\u00a8\u00e7\u00a0\u0094\u00e7\u00a9\u00b6", "create_time": "2023-02-18T01:54:04.516060" }, { "id": "61605a9b-eaea-4e39-9795-15b299863358", "title": "\u00e4\u00bd\u0093\u00e8\u0082\u00b2\u00e4\u00bf\u00b1\u00e4\u00b9\u0090\u00e9\u0083\u00a8\u00e7\u00a0\u0094\u00e7\u00a9\u00b6", "create_time": "2023-02-18T01:52:26.265229" }, { "id": "602a3b1f-503a-45f2-aa60-6b1ef3a6a20d", "title": "\u00e7\u0094\u00a8Visio\u00e7\u0094\u00bb\u00e7\u00bb\u0093\u00e6\u009e\u0084\u00e5\u009b\u00be", "create_time": "2023-02-17T11:38:29.675274" }, { "id": "93baf991-8529-44f7-a43d-01f9b5155a54", "title": "\u00e8\u0087\u00aa\u00e5\u008a\u00a8\u00e5\u008c\u0096\u00e6\u0095\u00b0\u00e5\u0080\u00bc\u00e6\u00b0\u00b4\u00e6\u00b1\u00a0\u00e8\u00af\u0095\u00e9\u00aa\u008c\u00e7\u00b3\u00bb\u00e7\u00bb\u009f", "create_time": "2023-02-17T11:19:21.074599" }, { "id": "f920f328-56a4-48f2-8e37-b544b67c8f67", "title": "Data Preprocessing Module.", "create_time": "2023-02-17T10:06:25.118011" }, { "id": "9198d639-ac1a-4eb7-951e-ff99bb759170", "title": "\u00e4\u00bd\u00bf\u00e7\u0094\u00a8NDK\u00e6\u008e\u00a8\u00e9\u0080\u0081\u00e8\u00a7\u0086\u00e9\u00a2\u0091\u00e6\u00b5\u0081\u00e3\u0080\u0082", "create_time": "2023-02-17T09:18:38.777665" }, { "id": "7bfa0b45-cecf-4a8d-b81b-6f78d2411511", "title": "\u00e8\u00b0\u00b7\u00e6\u00ad\u008c\u00e6\u00b5\u008f\u00e8\u00a7\u0088\u00e5\u0099\u00a8\u00e6\u0088\u00aa\u00e5\u009b\u00be\u00e6\u008f\u0092\u00e4\u00bb\u00b6", "create_time": "2023-02-17T07:17:03.053739" }, { "id": "6a91d776-3ce1-4703-af6c-b4e03eec276a", "title": "\u00e6\u009b\u00b4\u00e6\u0096\u00b0QOpenGLTexture\u00e7\u00ba\u00b9\u00e7\u0090\u0086", "create_time": "2023-02-17T07:13:28.160207" }, { "id": "fa41e935-4549-4f35-a38f-a59154459ea0", "title": "\u00e7\u0094\u00a8\u00e6\u0088\u00b7\u00e8\u00af\u00b7\u00e6\u00b1\u0082\u00e5\u00b8\u00ae\u00e5\u008a\u00a9", "create_time": "2023-02-17T06:25:01.085311" }, { "id": "c1671ab3-a9d1-4bd0-b817-8f29e7f64a22", "title": "OpenGL\u00e8\u00b4\u00b4\u00e5\u009b\u00be\u00e6\u0094\u00af\u00e6\u008c\u0081\u00e9\u0080\u008f\u00e6\u0098\u008ePNG", "create_time": "2023-02-17T03:04:40.888138" }, { "id": "c65b1753-c917-41f2-a69a-123f56fa1616", "title": "\u00e5\u00a4\u009a\u00e7\u009c\u008b\u00e4\u00b8\u0080\u00e7\u009c\u00bc\u00e7\u0088\u0086\u00e7\u0082\u00b8", "create_time": "2023-02-17T01:11:41.350951" }, { "id": "b42a37fd-4649-450a-bbce-64d808135d32", "title": "5\u00e8\u00bd\u00ae\u00e6\u0092\u00ad\u00e5\u009b\u00be\u00e5\u0092\u008c\u00e5\u0088\u0087\u00e6\u008d\u00a2\u00e6\u008c\u0089\u00e9\u0092\u00ae", "create_time": "2023-02-16T15:00:35.446229" }, { "id": "4b2f0167-c2b8-4d94-b86e-fd713e85554d", "title": "New chat", "create_time": "2023-02-16T14:58:04.793101" }, { "id": "5f88c68e-2e5c-4eb5-9ae5-c86a7ff7f6fd", "title": "\u00e7\u008c\u009c\u00e4\u00ba\u00ba\u00e7\u009a\u0084\u00e8\u00af\u00b7\u00e6\u00b1\u0082", "create_time": "2023-02-16T07:11:20.680333" }, { "id": "1ffe1d2c-827a-42ed-9986-66d40aa2f4fa", "title": "\u00e5\u008a\u00a8\u00e6\u00bc\u00ab\u00e4\u00ba\u00ba\u00e7\u0089\u00a9\u00e5\u009b\u00be\u00e7\u0089\u0087\u00e8\u00af\u00b7\u00e6\u00b1\u0082", "create_time": "2023-02-16T04:37:45.703397" }, { "id": "cd48a15d-4fd7-4f4e-99d5-12b7274f6400", "title": "ChatGPT\u00e8\u0083\u00bd\u00e5\u00b8\u00ae\u00e5\u0088\u00b0\u00e5\u0090\u0097\u00ef\u00bc\u009f (Can ChatGPT Help?)", "create_time": "2023-02-16T04:08:43.099001" }, { "id": "332ce738-23cf-45ef-9268-8dace9816079", "title": "5-image carousel website", "create_time": "2023-02-16T04:04:41.115613" }, { "id": "7eef9f92-7bbe-4080-9f30-6a4301ff33aa", "title": "\u00e6\u0099\u00ba\u00e8\u0083\u00bd\u00e6\u00a8\u00a1\u00e5\u009e\u008b\u00e7\u009a\u0084\u00e8\u0083\u00bd\u00e5\u008a\u009b", "create_time": "2023-02-16T04:03:21.610758" }, { "id": "4bd8951f-1432-4665-9237-72baa2e94ea6", "title": "\u00e7\u0094\u00a8\u00e6\u0088\u00b7\u00e5\u00af\u00bb\u00e6\u00b1\u0082\u00e5\u00b8\u00ae\u00e5\u008a\u00a9", "create_time": "2023-02-16T04:02:55.448398" }, { "id": "b451a201-18bc-4c17-90e2-4ce966909882", "title": "Android \u00e5\u0088\u0087\u00e6\u008d\u00a2\u00e8\u00a7\u0086\u00e5\u009b\u00be\u00e5\u00b1\u0082\u00e7\u00ba\u00a7\u00e3\u0080\u0082", "create_time": "2023-02-15T12:21:42.266015" }, { "id": "7c36fa57-7b11-4b00-83a4-26031fd4ff8d", "title": "\u00e8\u00ae\u00a1\u00e7\u00ae\u0097\u00e7\u0085\u00a7\u00e7\u0089\u0087\u00e4\u00b8\u00ad\u00e7\u009a\u0084\u00e7\u0082\u00b9", "create_time": "2023-02-15T09:02:25.910541" }, { "id": "d2ff0a10-c02c-43e8-8e14-df6f865ecb4d", "title": "\u00e8\u00ae\u00a1\u00e7\u00ae\u0097\u00e7\u00bb\u008f\u00e7\u00ba\u00ac\u00e5\u00ba\u00a6\u00e8\u00b7\u009d\u00e7\u00a6\u00bb\u00e5\u0092\u008c\u00e8\u00a7\u0092\u00e5\u00ba\u00a6 (Calculate Distance & Angle)", "create_time": "2023-02-15T07:49:23.931461" }, { "id": "47b4a5ff-94fc-476c-b550-9eb143cd4c11", "title": "\u00e6\u0097\u00a5\u00e6\u009c\u009f\u00e6\u009f\u00a5\u00e8\u00af\u00a2", "create_time": "2023-02-15T04:25:55.024948" } ]

脚本头加了utf8编码定义
# -*- coding: UTF-8 -*-
from revChatGPT.V1 import Chatbot

**Comment by bynhack** [comment]
问题是发生在get_conversations和get_msg_history这两个接口的返回，ask接口正常

**Comment by rakuyoMo** [comment]
一样的问题，之前在 [discussions 794](https://github.com/acheong08/ChatGPT/discussions/794) 这里有过讨论，楼主你是什么版本的 Python？

顺带一提我使用的还是 2.2.7 版本的 `revChatGPT`，域名使用的还是 `https://chat.duti.tech/`，不清楚最新版本可不可以。

**Reply by bynhack** [reply]
Python版本最新的，脚本版本也是最新的，如果不同版本都是有这个问题，那只能怀疑那个代理服务的问题了

**Reply by rakuyoMo** [reply]
升级到最新的 [2.3.4](https://github.com/acheong08/ChatGPT/releases/tag/2.3.4) 版本之后解决了。
